play-json/src/test/scala/com/lightbend/lagom/scaladsl/playjson/PlayJsonSerializerSpec.scala

```diff
@@ -118,6 +118,41 @@ class PlayJsonSerializerSpec extends WordSpec with Matchers {
       }
     }
 
+    "pick up serializers for registry with migration" in withActorSystem(TestRegistry3) { system =>
+
+      val migratedEvent = MigratedEvent(addedField = 2, newName = "some value")
+
+      val serializeExt = SerializationExtension(system)
+      val serializer = serializeExt.findSerializerFor(migratedEvent).asInstanceOf[SerializerWithStringManifest]
+
+      val bytes = serializer.toBinary(migratedEvent)
+      val manifest = serializer.manifest(migratedEvent)
+
+
+      manifest shouldEqual classOf[MigratedEvent].getName + "#5"
+
```

Comment:
Thank you Ignasi for the careful review.
I made changes according to your suggestions. The only thing is that I cannot simply use `serializer.manifest(migratedEvent)` for the original test cases, as they are meant to mimick the manifest of the old unversioned event. Nevertheless, I did extract all these expected formatting in one place.